examples/bench_cudamat.py

```diff
@@ -1,11 +1,12 @@
+from __future__ import print_function, division
```

Comment:
Python <2.6 compatibility is not important, I just wouldn't want to drop it for no reason. But I guess it's hard to have printing without newlines work in both Python 2 and 3 without importing `print_function`, so I'm fine with making the benchmark script require Python 2.6+. For the examples, however, you can just remove the `print_function` import and it will still work in both Python 2 and 3 (using the fact that `print(x)` is the same as `print (x)` is the same as `print x` in Python 2). That would be welcome!